alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultConfigurationAccessor.java

```diff
@@ -356,21 +366,27 @@ private ConfigurationModel createConfigForRelevantFields(final String descriptor
         return createConfiguration(descriptorName, ConfigContextEnum.DISTRIBUTION, relevantFields);
     }
 
-    private List<ConfigurationModel> createConfigModels(final Collection<RegisteredDescriptorEntity> descriptors) throws AlertDatabaseConstraintException {
+    private List<ConfigurationModel> createConfigModels(Collection<RegisteredDescriptorEntity> descriptors) throws AlertDatabaseConstraintException {
         final List<ConfigurationModel> configs = new ArrayList<>();
         for (final RegisteredDescriptorEntity descriptorEntity : descriptors) {
             final List<DescriptorConfigEntity> descriptorConfigEntities = descriptorConfigsRepository.findByDescriptorId(descriptorEntity.getId());
             for (final DescriptorConfigEntity descriptorConfigEntity : descriptorConfigEntities) {
-                final ConfigurationModel newModel = createConfigModel(descriptorConfigEntity.getDescriptorId(), descriptorConfigEntity.getId(), descriptorConfigEntity.getContextId());
+                final ConfigurationModel newModel = createConfigModel(descriptorConfigEntity.getDescriptorId(), descriptorConfigEntity.getId(),
+                    descriptorConfigEntity.getCreatedAt(), descriptorConfigEntity.getLastUpdated(), descriptorConfigEntity.getContextId());
                 configs.add(newModel);
             }
         }
         return configs;
     }
 
-    private ConfigurationModel createConfigModel(final Long descriptorId, final Long configId, final Long contextId) throws AlertDatabaseConstraintException {
+    private ConfigurationModel createConfigModel(Long descriptorId, Long configId, Date createdAt, Date lastUpdated, Long contextId) throws AlertDatabaseConstraintException {
         final String configContext = getContextById(contextId);
-        final ConfigurationModel newModel = new ConfigurationModel(descriptorId, configId, configContext);
+
+        SimpleDateFormat formatter = new SimpleDateFormat(DATE_FORMAT);
```

Comment:
We use SimpleDateFormat in multiple places.  I think that should be a separate PR to consolidate the usage of SimpleDateFormat because we have a few different format strings.